src/cogs/auto_mod.py

```diff
@@ -61,7 +59,7 @@ async def is_duplicate(self, message):
 
                 if (
                     SequenceMatcher(None, message.content, candidate.content).ratio()
-                    > self.file["dupe_thresh"]
+                    > conf.dupe_thresh
                 ):
```

Comment:
Why the ()?

```py
if SequenceMatcher(...) >
   conf.dupe_thresh:
```

Reply:
The if condition has to be in a single line, otherwise it has to be enclosed in parentheses.